kafka-streams-samples/kafka-streams-recoverable/src/test/java/kafka/streams/record/recoverable/KafkaStreamsRecoverableSampleTest.java

```diff
@@ -0,0 +1,115 @@
+package kafka.streams.record.recoverable;
+
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.junit.AfterClass;
```

Comment:
I think since this is a new sample, we should use JUnit 5.

Reply:
Updated in 391f70fbd7ddef62de22c216276e38b246969f9e